source/Halibut.Tests/Support/SerilogLoggerBuilder.cs

```diff
@@ -52,7 +52,7 @@ public void Emit(LogEvent logEvent)
                 // This is the change, call this instead of: TestContext.Progress
 
                 var logLine = output.ToString();
-                if (TeamCityDetection.IsRunningInTeamCity())
+                if (TeamCityDetection.IsRunningInTeamCity() || !string.IsNullOrWhiteSpace(Environment.GetEnvironmentVariable("Force_Test_Context_Write")))
```

Comment:
Force write if Progress doesn't work for your ide of choice!!

Reply:
I find it works well in rider in Halibut, but in Tentacle Progress seems to stop working.

Reply:
This I think is the right change, we may even want to print some "if your seeing this message but no others then enable `Force_Test_Context_Write`" message to show once written by `TestContext.Write`. In another PR/nice to have/sharpening.